docs/index.asciidoc

```diff
@@ -110,12 +119,10 @@ filter plugins.
 ===== `destination`
 
```

Comment:
```suggestion
deprecated[3.3.0]
```

Reply:
This is how we format deprecation notices. Looks like this: 
<img width="860" alt="Screen Shot 2021-06-22 at 3 13 16 PM" src="https://user-images.githubusercontent.com/35154725/122985656-70f58780-d36c-11eb-829d-6acbfec1ac89.png">

Here's another option. 
`deprecated[3.3.0, The default value has been changed to `false`. In 8.0.0 this setting will be removed]`
It gives you a place to put more info, but it doesn't look as good (because it renders bigger than the heading). Looks like this: https://www.elastic.co/guide/en/logstash/current/plugins-inputs-beats.html#plugins-inputs-beats-add_hostname.

Reply:
Thanks, I've added the *deprecated* marker. 
There's also a new `source` option (as a replacement for the now deprecated `field`), let me know if I should tune the deprecation warning further ... :cake:

Reply:
Your deprecation notices look like this: <img width="621" alt="Screen Shot 2021-06-24 at 4 45 53 PM" src="https://user-images.githubusercontent.com/35154725/123330223-c0be8500-d50b-11eb-888c-826f5308501e.png">

Adding a link to the option they should use instead is a helpful. ❤️